api/src/main/java/org/openmrs/api/impl/UserServiceImpl.java

```diff
@@ -350,14 +350,16 @@ public Role saveRole(Role role) throws APIException {
 	 */
 	@Deprecated
 	public void changePassword(User u, String pw) throws APIException {
-		dao.changePassword(u, pw);
+		changePassword(u, "test", pw);
```

Comment:
i did but it was temporally. I was intending to ask you if it was possible to get a user's password given that i already have a User object. I used test because in my Unit test, i used the default autenticated user who's password is test but i new this would not be okay because if in feature this password changes, my unit test will break also, if that changePassword method is called with a user who's password is not test will cause the code to fail. So my question now is, given that i have a user object, how can i retrieve his password as a string. Thanks